CMake/ctest_script.cmake

```diff
@@ -173,10 +180,14 @@ SET( CTEST_NIGHTLY_START_TIME "22:00:00 EST" )
 SET( CTEST_COMMAND "\"${CTEST_EXECUTABLE_NAME}\" -D ${CTEST_DASHBOARD}" )
 IF ( BUILD_SERIAL )
     SET( CTEST_BUILD_COMMAND "${MAKE_CMD} -i" )
+ELSEIF ( DEFINED N_PROCS_BUILD )
+    SET( CTEST_BUILD_COMMAND "${MAKE_CMD} -i -j ${N_PROCS_BUILD}" )
```

Comment:
No more need to test N_PROCS_BUILD. always use it.

Reply:
N_PROCS_BUILD is a non-default override.

Reply:
If we have always defined N_PROCS_BUILD properly. It is cleaner to have a single
``SET( CTEST_BUILD_COMMAND "${MAKE_CMD} -i -j ${N_PROCS_BUILD}" )``